Task 2/Task 2.1/Task 2.1/firure/Ring.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+
+namespace Task_2._1
+{
+    public class Ring : Figure , IAeraing
+    {
+        private Circle _inner;
+        private Circle _outer;
+        private double _sumR;
+
+        public double SumR
+        {
+            get { return _inner.R + _outer.R; }
+        }
+        public double S
+        {
+            get { return _inner.S - _outer.S; }
+        }
+
+        public Ring(Position centr, float innerR, float outerR) : base(centr,"Кольцо")
+        {
+            if (innerR > outerR)
+            {
+                throw new ArgumentException("innerR>outerR");
+            }
```

Comment:
Good, but message should describe what has gone wrong, like "inner radius can't be greater than outer radius"